content/recipes/swagger.md

```diff
@@ -77,75 +73,179 @@ async create(@Body() createCatDto: CreateCatDto) {
 }
 ```
 
-> warning **Notice** To implicitly set the body definition you can use the `@ApiImplicitBody()` decorator (`@nestjs/swagger` package).
+> info **Hint** To explicitly set the body definition use the `@ApiBody()` decorator (`@nestjs/swagger` package).
 
 Based on the `CreateCatDto`, the module definition will be created:
 
 <figure><img src="/assets/swagger-dto.png" /></figure>
 
-As you can see, the definition is empty although the class has a few declared properties. In order to make the class properties accessible to the `SwaggerModule`, we have to mark all of them with `@ApiModelProperty()` decorator:
+As you can see, the definition is empty although the class has a few declared properties. In order to make the class properties visible to the `SwaggerModule`, we have to either annotate them with `@ApiProperty()` decorator or use a CLI plugin (read more in the **Plugin** section):
```

Comment:
we have to either annotate them with `@ApiProperty()`  ->
we have to either annotate them with the `@ApiProperty()`  <- add **the**